clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerMembershipManagerTest.java

```diff
@@ -1617,12 +1617,25 @@ public void testRevokePartitionsUsesTopicNamesLocalCacheWhenMetadataNotAvailable
     }
 
     @Test
-    public void testOnSubscriptionUpdatedTransitionsToJoiningOnlyIfNotInGroup() {
+    public void testOnSubscriptionUpdatedDoNothingIfInGroup() {
```

Comment:
Thanks for the suggestion. I think whether the member is in the group is important as well, because it determines whether to change `shouldTransitionToJoining` variable. I update the test case name to `testOnSubscriptionUpdatedDoesNotSetShouldTransitionToJoiningIfInGroup` and another test case below to `testOnSubscriptionUpdatedSetShouldTransitionToJoiningIfNotInGroup`. Do you think it makes sense? Thank you.